pkgs/applications/misc/candle/default.nix

```diff
@@ -5,12 +5,14 @@ mkDerivation rec {
   version = "1.1";
 
   src = fetchFromGitHub {
-    owner  = "Denvi";
-    repo   = "Candle";
-    rev    = "v${version}";
+    owner = "Denvi";
+    repo = "Candle";
+    rev = "v${version}";
     sha256 = "1gpx08gdz8awbsj6lsczwgffp19z3q0r2fvm72a73qd9az29pmm0";
   };
 
+  patches = [ ./candle-settings.diff ];
```

Comment:
Please include comment on why we have this patch. Mention that upstream has gone quiet, etc...

Reply:
@aanderse Sure.  Please forgive the newbie questions that follow -- I don't spend much time in Github PR Land. :) 
I don't think I can rebase a PR (can I?), so did you want me to git-revert my changes, and then make the new commits you indicated?  Or something else -- e.g., should each of the three changes be in its own separate PR?